shared/chat/conversation/input/index.desktop.js

```diff
@@ -19,16 +20,23 @@ type InputProps = {
   filePickerOpen: () => void,
   filePickerSetValue: (value: any) => void,
   filePickerSetRef: (r: any) => void,
-  mentionPopupOpen: boolean,
-  setMentionPopupOpen: (setOpen: boolean) => void,
+  channelMentionFilter: string,
+  channelMentionPopupOpen: boolean,
+  setChannelMentionFilter: (filter: string) => void,
+  setChannelMentionPopupOpen: (setOpen: boolean) => void,
   mentionFilter: string,
+  mentionPopupOpen: boolean,
   setMentionFilter: (filter: string) => void,
+  setMentionPopupOpen: (setOpen: boolean) => void,
 } & Props
 
 type State = {
   upArrowCounter: number,
   downArrowCounter: number,
   pickSelectedCounter: number,
+  channelUpArrowCounter: number,
```

Comment:
I think you could reuse the three counters for both huds, since their behavior is pretty much identical & all their behavior is predicated on `channelMentionPopupUpen` & `mentionPopupOpen`